pkg/generate/ack/apis.go

```diff
@@ -105,5 +106,6 @@ type templateAPIVars struct {
 // code for a single top-level resource's API definition
 type templateCRDVars struct {
 	templateset.MetaVars
-	CRD *ackmodel.CRD
+	SDKAPI *ackmodel.SDKAPI
```

Comment:
You might consider adding this to the "root" `templateset.MetaVars` at some point in the future.